web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -339,10 +339,12 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
 
     _onTimeSelected: function(datePicker, selectedDateTimeMoment) {
         datePicker.setValue(selectedDateTimeMoment);
+        datePicker.tf.collapse();
```

Comment:
Time dropdown doesn't collapse for me when I select a time as per problem description - or was this for something else?